File: atlas/examples/kotlin/lib/src/main/kotlin/dev/atlas/example/lib/ReadingListItem.kt
package dev.atlas.example.lib

/**
 * Represents one normalized book title and its stable URL-safe slug.
 *
 * @property title Normalized non-empty display title.
 * @property slug Stable lower-case hyphenated identifier.
 */
data class ReadingListItem(
    val title: String,
    val slug: String
)

File: atlas/examples/csharp/src/lib/ReadingListItem.cs
namespace Atlas.Example.Lib;

/// <summary>
/// Represents a validated book title and its stable normalized slug.
/// </summary>
/// <param name="Title">The normalized display title.</param>
/// <param name="Slug">The lowercase, hyphen-delimited stable identifier.</param>
public sealed record ReadingListItem(string Title, string Slug);
